content/Assets/Scripts/components/toggleNav/index.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Initialises the toggle nav button that's responsible for showing/hiding
+ * the main menu (likely on smaller devices).
+ */
+const toggleNav = () => {
+    const CSS_VISIBLE = 'is-nav-visible';
+    const SELECTOR = '.js-nav-button';
+
+    const validHideKeyCodes = [38];
+    const validShowKeyCodes = [40, 32];
```

Comment:
Not entirely relevant to the purpose of this update but it might be nice if these key codes used a constant to state which keys they are for.